manifests/users.pp

```diff
@@ -6,14 +6,32 @@
   $metrics_port   = 2003,
   $daemonize      = false,
   $run_pxp        = true,
+  $use_cached_catalog = $clamps::agent::use_cached_catalog,
+  $run_interval   = $clamps::agent::run_interval,
   $splay          = false,
   $splaylimit     = undef,
 ) {
 
-  $user_cron_minute = clamps_user_number($user) % 30
-
-  $cron_1 = $user_cron_minute
-  $cron_2 = $user_cron_minute + 30
+  if $run_interval == 30 {
```

Comment:
Yeah. I want to make something better for turning an interval into a cron config. This was my hack for now.